app/views/common/_card_status_mini.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="<%= size %>">
+  <div class="card-pf card-pf-accented card-pf-aggregate-status card-pf-aggregate-status-mini">
+    <h2 class="card-pf-title">
+      <span class="<%= icon %>"></span>
```

Comment:
not sure why the icon is to the left, imho they could be next to the text/counter.